src/main/resources/assets/astralsorcery/lang/en_us.json

```diff
@@ -659,7 +660,7 @@
   "block.astralsorcery.infuser": "Starlight Infuser",
   "block.astralsorcery.lens": "Lens",
   "block.astralsorcery.observatory": "Observatory",
-  "block.astralsorcery.prism": "PrismLens",
+  "block.astralsorcery.prism": "Crystal Prism",
```

Comment:
Either name this just "Prism" or rename the Lens to "Crystal Lens"

Reply:
Either you forgot to tell Doomgull about this or idk. The prism is Crystal Prism in the new tome anyway. Same with spectral relays now being astral relays.

Also, PrismLens sounds like something you would find at a shelf in a store, too product-y. It's just a prism made out of crystal.

Reply:
Oh, I get what you mean. I fully support renaming the Crystal Lens to differentiate it from the glass lenses.

Reply:
addressed